src/dpp/message.cpp

```diff
@@ -249,6 +249,8 @@ void to_json(json& j, const attachment& a) {
 		j["url"] = a.url;
 	}
 	j["ephemeral"] = a.ephemeral;
+	/* in future, Discord can allow bots send remixed attachment */
+	j["flags"] = a.flags;
```

Comment:
where do you hear this from? we can't just add support on heresay as poeple will try to use it, at worst case sending flags when it is not supported will generate an error.

Reply:
calm down, Discord API ignores 'flags' field if bot account, using user account would result attachment with flags

Reply:
still, if the docs don't have it we shouldn't send it

Reply:
Ok fine